config.json

```diff
@@ -2,5 +2,9 @@
     "defaultVersion": "0.3.5-alpha",
```

Comment:
Consider making "0.4.0-alpha" the default version in non-production settings.  If you choose to do this (and I'm not saying you should) I don't know if it's possible to automatically select the last entry from the "visibleVersions" or "allVersions" list, or if it would have to be selected according to the environment variable.